src/components/shared/SearchInput.js

```diff
@@ -39,6 +39,10 @@ class SearchInput extends Component {
 
   static defaultProps: Object;
 
+  componentDidMount() {
+    this.$input.focus();
```

Comment:
Because the modal is mounted the symbol modal needs to have that if block to focus the input. I'll research further to see if there's a solution that's cleaner but this gets the focus events working on all the search dialogs and ensures we have no regressions on the shortcuts.